### DISCUSSION TITLE
PyPI update

### DISCUSSION THREAD

**Discussion by codedust** [latchset/jwcrypto#265] [General]
Hi! Thanks for this great library!

I would really like to use version 1.1 using pip/poetry. Could you update https://pypi.org/project/jwcrypto/ to the latest version?

Thanks in advance!

**Comment by simo5** [comment]
Unfortunately I do not have access to pip at the moment.

@tiran might you have so me time?

**Comment by codedust** [comment]
Friendly ping :slightly_smiling_face:

**Comment by codedust** [comment]
Thanks for the update! :blush:

**Comment by simo5** [comment]
https://github.com/latchset/jwcrypto/releases/tag/v1.2.0
